### PR TITLE
Moving LMS loadtest to the public repo.

### PR DESCRIPTION
**Pull request by feanil**
Also, I had to add a second '-e' requirement to the base.txt for opaque-keys
since that repo's versions are not properly managed right now.

@e0d  @benpatterson @nedbat @jbarciauskas

### CODE REVIEW

**Comment by feanil** [comment]
I'll fix the PEP* violations when I have some free time but if one of you has bandwidth please feel free to push the fixes onto this branch.

**Comment by e0d** [comment]
:+1: pending pylint

**Comment by benpatterson** [comment]
I've fixed the violations, though, I had to ignore a rule to do so. The violation we're ignoring is:
`lms/lms.py:13:1: E402 module level import not at top of file`. 

It's hacky to begin with, so I don't feel great about disabling it. However, I don't have time to ensure a change to the file will/will-not impact how locust is run, so I'm leaving it. Someone else can feel free to tackle that one or we can leave it as a TODO.

**Comment by feanil** [comment]
@benpatterson @nedbat second thumb?

**Comment by benpatterson** [comment]
Oh. :+1:

File: requirements/base.txt [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Packages required to run load tests
+# Using -e for now because neither of these packages have released code
+# properly.  Once they have done releases with actualy versions we should
+pin these packages correctly.
```
Comment (nedbat) [inline requirements/base.txt]: This line should have a comment marker, right?

**Pull request merged**